src/EditorFeatures/CSharpTest/Workspaces/WorkspaceTests.cs

```diff
@@ -28,7 +28,7 @@ namespace Microsoft.CodeAnalysis.UnitTests.Workspaces
     public partial class WorkspaceTests : TestBase
     {
         private static TestWorkspace CreateWorkspace(string workspaceKind = null, bool disablePartialSolutions = true)
-            => new TestWorkspace(TestExportProvider.ExportProviderWithCSharpAndVisualBasic, workspaceKind, disablePartialSolutions);
+            => new TestWorkspace(TestExportProvider.ExportProviderWithCSharpAndVisualBasic, composition: null, workspaceKind, disablePartialSolutions);
```

Comment:
💭 I'd love to see `TestExportProvider.ExportProviderWithCSharpAndVisualBasic` go away in favor of a standard scenario composition

Reply:
Yup, getting there :)